### ISSUE TITLE
Completed docs scheme seems to be incorrect.

### ISSUE THREAD

**Issue by RedTailBullet** [palantir/tslint#4185]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 3.0.3
- __Running TSLint via__: (pick one) VSCode

#### TypeScript code being linted

with `tslint.json` configuration:

```json
        "completed-docs": {
            "options": {
                "functions": {
                    "visibilities": "exported"
                },
                "methods": {
                    "privacies": [
                        "protected",
                        "public"
                    ]
                },
                "properties": {
                    "privacies": [
                        "protected",
                        "public"
                    ]
                }
            }
        }
```

#### Actual behavior

The arrays for privacies in methods and properties will cause an warning being reported as it only accept string.

#### Expected behavior

According to the example provided by the rule page, this seeting should not throw any warning.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
